config/initializers/sentry.rb

```diff
@@ -0,0 +1,26 @@
+Sentry.init do |config|
+  # Sentry config here is mostly copied from apply-for-teacher-training
+  # https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/config/initializers/sentry.rb
+  config.environment = ActiveSupport::EnvironmentInquirer.new(ENV.fetch("HOSTING_ENV", "development"))
```

Comment:
I think this should work? 🤔

```suggestion
  config.environment = HostingEnvironment.env
```